go/staking/api/api.go

```diff
@@ -742,6 +743,7 @@ const (
 	KindNodeKeyManagerName    = "node-keymanager"
 	KindRuntimeComputeName    = "runtime-compute"
 	KindRuntimeKeyManagerName = "runtime-keymanager"
+	KindKeyManagerChurpName   = "keymanager-churp"
 )
 
 // ThresholdKinds are the valid threshold kinds.
```

Comment:
Add comment in place of `KindKeyManagerChurpName` explaining why it is not there.